File: despatch.WEB/app/src/swagger/models/user-request.ts
/* tslint:disable */
import { IdentityUser } from './identity-user';
import { UserAddressesRequest } from './user-addresses-request';
import { CityRequest } from './city-request';
import { CountryRequest } from './country-request';
import { UserImagesRequest } from './user-images-request';
import { UserOrganizationsRequest } from './user-organizations-request';
import { UserPhoneNumbersRequest } from './user-phone-numbers-request';
import { UserUrlsRequest } from './user-urls-request';
export interface UserRequest extends IdentityUser{
  addresses?: Array<UserAddressesRequest>;
  city?: CityRequest;
  cityId?: string;
  country?: CountryRequest;
  countryId?: string;
  creationDate: string;
  images?: Array<UserImagesRequest>;
  organizations?: Array<UserOrganizationsRequest>;
  phoneNumbers?: Array<UserPhoneNumbersRequest>;
  urls?: Array<UserUrlsRequest>;
}

File: despatch.Domain/Models/Default/Request/UserRequest.cs

using Microsoft.AspNetCore.Identity;
using System;
using System.Collections.Generic;

namespace despatch.Domain.Models.Default.Request
{
    public class UserRequest : IdentityUser
    {
        public DateTime CreationDate { get; set; }

        public string CityId { get; set; }
        public string CountryId { get; set; }

        public virtual CityRequest City { get; set; }
        public virtual CountryRequest Country { get; set; }

        public ICollection<UserAddressesRequest> Addresses { get; set; }
        public ICollection<UserOrganizationsRequest> Organizations { get; set; }
        public ICollection<UserImagesRequest> Images { get; set; }
        public ICollection<UserPhoneNumbersRequest> PhoneNumbers { get; set; }
        public ICollection<UserUrlsRequest> Urls { get; set; }

    }
}
